content/SCALETutorials/Instances/_index.md

```diff
@@ -228,29 +228,13 @@ To create a new container, from the **Create Instance** screen:
 
    d. Click **Add** again to mount additional storage volumes.
 
-5. (Optional) Configure proxy settings to forward network connections between the host and the container.
-   This routes traffic from a specific address on the host to an address inside the instance, or vice versa, allowing the instance to connect externally through the host.
+5. {{< include file="/static/includes/InstanceNetworkProcedure.md" >}}
 
-   {{< trueimage src="/images/SCALE/Virtualization/CreateInstanceProxies.png" alt="Proxies" id="Proxies" >}}
+6. {{< include file="/static/includes/InstanceUSBProcedure.md" >}}
 
-   a. Click **Add** in the **Proxies** section to display a set of proxy configuration settings.
+7. {{< include file="/static/includes/InstanceGPUProcedure.md" >}}
 
-   b. Select the protocol option from the **Host Protocol** dropdown list to set the connection protocol for the TrueNAS host as **TCP** or **UDP**.
-
-   c. Enter a port in **Host Port** to define the TrueNAS port to map to the instance port on the container, for example *3600*.
-
-   d. Select the connection protocol for the container in **Instance Protocol**.
-      Options are **TCP** or **UDP**.
-
-   e. Enter the port number within the container in **Instance Port**, for example *80*, to map to the host port.
-
-6. {{< include file="/static/includes/InstanceNetworkProcedure.md" >}}
-
-7. {{< include file="/static/includes/InstanceUSBProcedure.md" >}}
-
-8. {{< include file="/static/includes/InstanceGPUProcedure.md" >}}
-
-9. Click **Create** to deploy the container.
+8. Click **Create** to deploy the container.
 
 ### Creating a Virtual Machine
```

Comment:
In step 1. c. below the substep numbering is off. It should go back to 1,2,3 not a,b,c again.
Also, the two bullet points for the options associated with this step read like a step. Please reword to sound less like an action step to something about the option of using it. Maybe just change "select" to "Selecting **Use a Linux Image** creates the VM using a Linux image from ...."  and finishing up the sentence as written. Same for that other bullet point. A small change but it reads less like an action step that should be numbered. :-)